contrib/clojure-package/README.md

```diff
@@ -80,36 +86,33 @@ wget https://archive.archlinux.org/packages/c/cuda/cuda-9.0.176-4-x86_64.pkg.tar
 sudo pacman -U cuda-9.0.176-4-x86_64.pkg.tar.xz
 ```
 
-If you want to see the exact versions and flags that the jars were built with, look here:
-[Scala Release Process](https://cwiki.apache.org/confluence/display/MXNET/MXNet-Scala+Release+Process)
+At this point you should be able to run your own example like this [NDArray Tutorial](https://github.com/apache/incubator-mxnet/blob/master/contrib/clojure-package/examples/tutorial/src/tutorial/ndarray.clj)
 
+### Getting Started with MXNet project with the Prebuilt Scala Jars
 
-Check your installation with `lein test`. If that works alright then, you can try some code!
+This option is also fast. It doesn't require you to build the native C library or the Scala jars, it gets them from [Maven](https://search.maven.org/search?q=g:org.apache.mxnet) as well. 
```

Comment:
It would be slightly redundant, but I think maybe a sentence here similar to the one under *Getting Started with the Clojure Jars on Maven* ("If you are looking to use MXNet in your own Clojure project, this is your best bet.") would provide confusion-alleviating context.

Perhaps "Use this if you want your Clojure project to use a version of MXNet that hasn't yet landed on Maven."—or perhaps some other use case unknown to me is more relevant.